core/src/com/gungame/world/objects/weapon/GunData.java

```diff
@@ -0,0 +1,65 @@
+package com.gungame.world.objects.weapon;
+
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.audio.Sound;
+import com.badlogic.gdx.graphics.Texture;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+@Getter
+@RequiredArgsConstructor
+public enum GunData {
+    RIFLE(0.035f,
+            20,
```

Comment:
Предлагаю немного повысить урон для баланса

```suggestion
            35,
```